shell/platform/fuchsia/flutter/platform_view.cc

```diff
@@ -96,19 +76,18 @@ PlatformView::PlatformView(
       await_vsync_for_secondary_callback_callback_(
           await_vsync_for_secondary_callback_callback),
       weak_factory_(this) {
-  // Register all error handlers.
-  SetInterfaceErrorHandler(ime_, "Input Method Editor");
-  SetInterfaceErrorHandler(ime_client_, "IME Client");
-  SetInterfaceErrorHandler(text_sync_service_, "Text Sync Service");
-  SetInterfaceErrorHandler(keyboard_listener_binding_, "Keyboard Listener");
-  SetInterfaceErrorHandler(keyboard_, "Keyboard");
-
   fuchsia::ui::views::ViewRef view_ref_clone;
   fidl::Clone(view_ref, &view_ref_clone);
 
-  // Configure keyboard listener.
-  keyboard_->AddListener(std::move(view_ref),
-                         keyboard_listener_binding_.NewBinding(), [] {});
+  text_delegate_ = std::make_unique<TextDelegate>(
```

Comment:
Since its not conditional, can this be created earlier, in the class initializer list?

Reply:
It used to be in the initializer list when I started working on it. But, in the meantime, a viewref clone operation was added, which I think means I can no longer move `view_ref` in the initializer list and preserve the correct semantics. So, this was the effect.

Reply:
I think you can reference it, make your own clone internally, and leave the original reference for other things to use?